MonopolyKata/Monopoly.cs

```diff
@@ -76,14 +84,44 @@ private void DetermineOrder()
             }
         }
 
-        public void RunMonopoly()
+        public void RunMonopoly(Int32 numberOfRounds)
         {
             var i = 0;
-            while (i < 20)
+            while (i < numberOfRounds)
             {
                 var j = 1;
-                while (j < 5){
-                    order[j].RollDicePair();
+                while (j < 5)
```

Comment:
A good rule of thumb is that the larger the scope of the variable the more descriptive its name should be. This loop has a good sized scope, therefore it would be helpful if "i" and "j" were more description.